rpm-4.8.0-htonll-fix.patch

```diff
@@ -0,0 +1,48 @@
+diff --git a/lib/header.c b/lib/header.c
```

Comment:
Yes.


Sent on the new Sprint Network from my Samsung Galaxy S®4.


-------- Original message --------
From: Giulio Eulisse
Date:05/16/2015 2:22 AM (GMT-06:00)
To: cms-sw/cmsdist
Cc: Patrick E Gartung
Subject: Re: [cmsdist] Changes to build on OSX Yosemite. (#1558)


In rpm-4.8.0-htonll-fix.patch<https://github.com/cms-sw/cmsdist/pull/1558#discussion_r30459217>:

> @@ -0,0 +1,48 @@
> +diff --git a/lib/header.c b/lib/header.c


I created https://github.com/cms-externals/rpm which has all the current patches but this one in the cms/v4.8.0 branch. Does this one work on slc6?

—
Reply to this email directly or view it on GitHub<https://github.com/cms-sw/cmsdist/pull/1558/files#r30459217>.